Sources_v3/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -190,6 +190,12 @@ public class _ChatChannelController<ExtraData: ExtraDataTypes>: DataController,
         client.webSocketClient,
         client.apiClient
     )
+
+    private lazy var messageUpdater: MessageUpdater<ExtraData> = self.environment.messageUpdaterBuilder(
+        client.databaseContainer,
+        client.webSocketClient,
+        client.apiClient
+    )
```

Comment:
Hmm, why did you decided to do this? It's not a pattern we usually use. Message-related methods belong to message controller 🤔

Reply:
As far as we work with the message that hasn't been sent yet I realized we cannot have a message controller for it. But we return `messageId` from `createNewMessage` so that's possible 👍 

moved to `ChatMessageController` in 32f17a83201c3be8a9f27ced7f0f096cac368f24